tests/scripts/task_python_integration.sh

```diff
@@ -55,6 +55,9 @@ TVM_FFI=ctypes python3 -m pytest apps/dso_plugin_module
 # TVM_FFI=ctypes sh prepare_and_test_tfop_module.sh
 
 TVM_FFI=ctypes python3 -m pytest tests/python/integration
+if python -c "import tvm; from tvm.relay.op.contrib.ethosn import ethosn_available; print(ethosn_available().name)" -eq "SW_ONLY"; then
+  ETHOSN_VARIANT_CONFIG=ETHOSN78_1TOPS_4PLE_448KSRAM TVM_FFI=ctypes python3 -m pytest tests/python/contrib/test_ethosn
```

Comment:
this logic also needs to be in tests/scripts/task_python_ethosn_tests.sh so that folks just running the tests for Ethos-N77 can get the tests clean

Reply:
Not needed for the test suite you identified as that one's not part of the docker image and already targets the ethosn tests specifically - It defaults to Ethos-N77 but setting ETHOSN_VARIANT_CONFIG appropriately switches that to Ethos-N78

Reply:
I see,  could you add a comment there to indicate this ?

Reply:
Done.